Sources/Document/CRDT/CRDTElement.swift

```diff
@@ -18,16 +18,25 @@ import Foundation
 
 /**
  * `CRDTElement` represents element type containing logical clock.
- *
- * @internal
  */
-class CRDTElement {
-    private var createdAt: TimeTicket
-    private var movedAt: TimeTicket?
-    private var removedAt: TimeTicket?
+protocol CRDTElement: AnyObject {
```

Comment:
Changed CRDTElement type from class to protocol for flexibility and stability.
Swift’s protocols are similar to Java’s interfaces.

Reply:
@hyeongsik-won Never know about protocol in Swift before. 👍